packages/checkout-com/nuxt/proxyMiddleware.js

```diff
@@ -0,0 +1,79 @@
+const axios = require('axios');
+const express = require('express');
+const app = express();
+
+let ckoPublicKey = null;
+let ckoSecretKey = null;
```

Comment:
rename because of conflict names, or wrap it inside the object:

```
const config {
 ckoPublicKey: null,
 ckoSecretKey
}
```

and use it always  as config.XXX

Reply:
const config = {
  publicKey: null,
  secretKey: null,
  ctApiUrl: null
}